defopt.py

```diff
@@ -158,7 +166,9 @@ class _NoTypeFormatter(_Formatter):
     show_types = False
 
 
-def _populate_parser(func, parser, parsers, short):
+def _populate_parser(func, parser, parsers, **kwargs):
+    short = kwargs['short']
+    strict_kwonly = kwargs['strict_kwonly']
```

Comment:
I don't understand why you would pass and receive arguments this way. What does do for you that the previous code didn't?